jpirumvaa/jpirumva
# #Gylh0t8A/35 Use DRY in Projects

#### What does this PR do?
The projects should display the completed projects. This PR removes unnecessary codes and uses JavaScript to make codes simple.
#### Description of Task to be completed?
Have Projects in both Homepage and Projects page using JavaScript to display Projects and avoiding repetition of tags and content.
#### How should this be manually tested?
To test this part, download the codes, and check in the text editor if Projects in the Homepage and Projects page are using JavaScript. Open them into the browser to see if all projects are displaying correctly. 
#### Any background context you want to provide?
There is a project repeated three times. The two are the placeholders for the projects I will upload soon.
#### What are the relevant pivotal tracker stories?
[https://trello.com/c/Gylh0t8A/35-refactor-products-to-use-dry]()
#### Screenshots (if appropriate)
![projects](https://user-images.githubusercontent.com/50476344/88680712-14083780-d0f1-11ea-8516-09938bff0253.png)
